cves/kernel/CVE-2017-17805.yml

```diff
@@ -55,7 +55,20 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
This is an amazing description of Salsa20 that is simple and easy to understand. Great Job! One this I would add though is what the consequence of exploiting the vulnerability would be other than that it crashes. If the whole system is crashing then specify that.

Reply:
Good thought!